lldb/source/Expression/IRExecutionUnit.cpp

```diff
@@ -279,6 +279,9 @@ void IRExecutionUnit::GetRunnableInfo(Status &error, lldb::addr_t &func_addr,
       .setMCJITMemoryManager(std::make_unique<MemoryManager>(*this))
       .setOptLevel(llvm::CodeGenOptLevel::Less);
 
+  if (triple.isRISCV64())
```

Comment:
This should have a comment. If every other arch set the code model it'd be fine, but this sticks out.

Reply:
addressed